docs/button.md

```diff
@@ -212,7 +212,7 @@ If `true`, disable all interactions for this component.
 
 ---
 
-### `hasTVPreferredFocus` <div className="label tv">TV</div>
+### `hasTVPreferredFocus` <div className="label ios">tvOS</div>
```

Comment:
In the TV repo, this prop is supported on both Apple TV and Android TV.

Reply:
Hey @douglowder, after discussing it with @cortinico, looks like want to keep the docs as close to the core repo code (no other out-of-tree platform specific props or prop changes are listed in the docs).

If you have a website or doc where we can redirect people interested in TV platforms let me know. We can add a link to the page which have a better/more detailed information for TV usage.

Also, if you think that those platform annotations are incorrect [in the core repo code](https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Components/Touchable/TouchableOpacity.d.ts#L18), please create a PR addressing that, and we can follow up with docs correction, if it gets merged.

Reply:
Also, if you need a help with setting up TV platforms website LMK, let's talk on Discord :)